flink-derived-feature-view/main.py

```diff
@@ -21,29 +21,31 @@
 from feathub.feature_views.transforms.over_window_transform import (
     OverWindowTransform,
 )
+from feathub.feature_views.transforms.join_transform import JoinTransform
```

Comment:
Is it added by accident? I can't find where it is used.

Reply:
Thanks for catching this. It is fixed now.